samples/openthread/coprocessor/README.rst

```diff
@@ -8,7 +8,7 @@ Thread: Co-processor
    :depth: 2
 
 The :ref:`Thread <ug_thread>` Co-processor sample demonstrates how to implement OpenThread's :ref:`thread_architectures_designs_cp` inside the Zephyr environment.
-Depending on the configuration, the sample uses the :ref:`thread_architectures_designs_cp_ncp` architecture or :ref:`thread_architectures_designs_cp_rcp` architecture.
+Depending on the configuration, the sample uses the :ref:`thread_architectures_designs_cp_rcp` architecture.
```

Comment:
```suggestion
The sample uses the :ref:`thread_architectures_designs_cp_rcp` architecture.
```
Alternatively, if it is not mandatory, it's "The sample can be configured to use the :ref:`thread_architectures_designs_cp_rcp` architecture."